test/dlc_tests/sanity/quick_checks/test_yml_config.py

```diff
@@ -0,0 +1,57 @@
+import os
+import re
+
+import pytest
+import yaml
+
+from test.test_utils import is_pr_context
+
+
+@pytest.mark.quick_checks
+@pytest.mark.model("N/A")
+@pytest.mark.integration("release_images_training.yml")
+@pytest.mark.skipif(
+    not is_pr_context(),
+    reason="This test is only needed to validate release_images configs in PRs.",
+)
+def test_release_images_training_yml():
+    _release_images_yml_verifier(image_type="training", excluded_image_type="inference")
+
+
+@pytest.mark.quick_checks
+@pytest.mark.model("N/A")
+@pytest.mark.integration("release_images_inference.yml")
+@pytest.mark.skipif(
+    not is_pr_context(),
+    reason="This test is only needed to validate release_images configs in PRs.",
+)
+def test_release_images_training_yml():
+    _release_images_yml_verifier(image_type="inference", excluded_image_type="training")
+
+
+def _release_images_yml_verifier(image_type, excluded_image_type):
+    """
```

Comment:
until neuron bug is fixed, should probably add a test condition to ensure that neuron and cpu/gpu are not defined for the same framework version